colibre/scripts/bh_timestep_hist.py

```diff
@@ -0,0 +1,169 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.interpolate import interp1d
+from matplotlib.lines import Line2D
+from glob import glob
+
+from swiftsimio import load
+import unyt
+
+bin_edges = np.logspace(np.log10(50e-3), np.log10(50e7), 41)  # kyr
```

Comment:
can we document this?

Reply:
And the rest of the timebin --> time-step algorithm in case someone else needs to edit this script in the future.